<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_studs_box</class>
 <widget class="QDialog" name="display_studs_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>812</width>
    <height>711</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enrolled students</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:#1F2937;
	color:white;
	font-weight:700;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
        font-size:35px;
        font-family:Arial, sans-serif;
  color:#ccc;
}</string>
   </property>
   <property name="text">
    <string>ENROLLED STUDENTS</string>
   </property>
  </widget>
  <widget class="QListWidget" name="studs_display_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>771</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background:#111827;;
	color:#CCC;
	font-weight:700;
	border-radius:11px;
    font-size: 20px;
}</string>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_all_studs_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>670</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		font-family:Arial, sans-serif;
		font-size:18px;
        font-weight:800;
        border-radius:12px;
        background:#881337;
}</string>
   </property>
   <property name="text">
    <string>delete all</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_studs_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    background:#888;
    border-radius:8px;
    color:#881337;
    font-size:17px;
}</string>
   </property>
   <property name="text">
    <string>Total : 100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
